engine/src/qlcfixturemode.h

```diff
@@ -206,9 +207,31 @@ class QLCFixtureMode
 
     quint32 masterIntensityChannel() const;
 
+    /*!
+     * \brief The ChannelActsOnData struct
+     *
+     * Contains channel pointer and acts on channel index.
+     *
+     */
+
+    struct ChannelActsOnData{
+        QLCChannel *channel;
+        int actsOnIndex;
+
+        ChannelActsOnData(QLCChannel *newChannel, int newAcsOnIndex) :
```

Comment:
I'm not sure this will work with older GCCs.
Might need to move it in the cpp file